liblangutil/Scanner.cpp

```diff
@@ -108,18 +108,18 @@ class LiteralScope
 		m_complete(false)
 	{
 		if (_type == LITERAL_TYPE_COMMENT)
-			m_scanner->m_nextSkippedComment.literal.clear();
+			m_scanner->m_skippedComments[Scanner::NextNext].literal.clear();
```

Comment:
Here it uses `Scanner::` and further down it doesn't.

Reply:
Yes, it's a different class.

Reply:
Ah ok, hadn't noticed